### PR TITLE
Update OWNERS and move google groups to its own dir

### PR DESCRIPTION
**Pull request by Bobgy**
/assign @jlewi @james-jwu @rmgogogo

### CODE REVIEW

File: OWNERS [hunk 1]
```diff
@@ -1,4 +1,8 @@
 approvers:
   - bobgy
+  - james-jwu
   - jlewi
-  - rmgogogo
\ No newline at end of file
+  - karkumar
```
Comment (jlewi) [inline OWNERS:5]:
```suggestion
```

Per #339 @karkumar wants to use @kramachandran that said lets remove @paveldournov and @karkumar as they likely won't actually be reviewing PRs to the google groups.

File: OWNERS [hunk 2]
```diff
@@ -1,4 +1,8 @@
...
+  - neuromage
+  - paveldournov
```
Comment (jlewi) [inline OWNERS:7]:
```suggestion
```

**Comment by Bobgy** [comment]
CLA failed because I used github suggestion feature

**Comment by Bobgy** [comment]
@rmgogogo Can you help me lgtm?

**Pull request merged**